python/tank/util/qt_importer.py

```diff
@@ -459,7 +418,7 @@ def _import_modules(self, interface_version_requested):
                 return pyside2
             except ImportError:
                 pass
-        elif interface_version_requested == self.QT5:
+        if interface_version_requested == self.QT5:
```

Comment:
Why this change? It looks wrong to me but maybe I'm missing something.

Reply:
you are right, that was because I was testing commenting the first condition block, but "elif" is the correct one